libs/client/client.ts

```diff
@@ -370,6 +370,29 @@ class Client {
       return null;
     }
   }
+
+  /**
+   *
+   * @function saveCommitments do the communications with commitments/saveAll
+   * endpoint
+   * @param listOfCommitments a list of commitments to be saved in the database.
+   * @author luizoamorim
+   */
+  async saveCommitments(listOfCommitments: Commitment[]) {
```

Comment:
Please, add a return (e.g. `boolean`) to this function to signal if things went well.

Reply:
Now I'm thinking. Is not better throw an error if some problem happens and not return nothing. Because if any error happens, so the flow is good.
@imagobea what do you think?

Reply:
So on top of the file we have now an axios interceptor. This will take care of client errors. It means that you don't need the `try/catch` block and that you don't need to worry about logging errors. Please take a look at the `get-contract-address` endpoint. You can just reproduce that.

You can also enhance the JSDocs. Replace "function" with `@method`. Add `@throws`, `@returns`.